agents-manage-ui/src/components/icons/google.tsx

```diff
@@ -0,0 +1,26 @@
+import { forwardRef } from 'react';
+import { type IconComponentProps, SvgIcon } from '@/components/ui/svg-icon';
+
+export const GoogleIcon = forwardRef<SVGSVGElement, IconComponentProps>((props, ref) => (
+  <SvgIcon ref={ref} {...props} strokeWidth={0}>
+    <title>Google icon</title>
```

Comment:
Shocked this doesn't come from an icon package. We should probably consider using one of the lucide-react icons if they are available.